feature-wallet-impl/src/main/java/jp/co/soramitsu/wallet/impl/presentation/balance/detail/BalanceDetailsScreen.kt

```diff
@@ -92,22 +94,28 @@ fun BalanceDetailsScreen(
 
     val scope = rememberCoroutineScope()
     val keyboardController = LocalSoftwareKeyboardController.current
+    val screenCallback = object : ChainSelectContentInterface {
```

Comment:
Why don't you just implemented ChainSelectContentInterface in ViewModel and pass the viewModel to the ChainSelectContent(callback = viewModel